libs/core/langchain_core/runnables/base.py

```diff
@@ -396,6 +396,30 @@ def __ror__(
         """Compose this runnable with another object to create a RunnableSequence."""
         return RunnableSequence(first=coerce_to_runnable(other), last=self)
 
+    def pick(self, keys: Union[str, List[str]]) -> Runnable[Any, Any]:
```

Comment:
what if this was `*keys: str`?

Reply:
A single key gets you the value itself whereas a list of key(s) gets you a dict with those keys

Reply:
could do `(self, *keys: str, key: Optional[str]=None)`?

Reply:
separately should we support `name: Optional[str]=None` and pass that to RunnableAssign?

Reply:
or just pass through **kwargs